doc/build.md

```diff
@@ -6,15 +6,15 @@ See also [Istio Development](https://github.com/istio/istio/blob/master/devel/RE
 
 We are using [Bazel 0.5.2](https://github.com/bazelbuild/bazel/releases) as the main build system in Istio Pilot. The following command builds all targets in Istio Pilot:
 
-To avoid a `missing input file '//platform/kube:config'` error you need to `ln -s ~/.kube/config platform/kube/`, see [testing](testing.md) for details/why.
+To avoid a `missing input file '//platform/kube:config'` error you need to `ln -s ~/.kube/config platform/kube/` or `make platform/kube/config`, see [testing](testing.md) for details/why.
```

Comment:
Nit. make kubeconfig. I don't see a change in makefike? Also this target can be a dependency on make setup or build